src/backend/distributed/commands/vacuum.c

```diff
@@ -299,6 +320,29 @@ DeparseVacuumStmtPrefix(int vacuumFlags)
 		appendStringInfoString(vacuumPrefix, "VERBOSE,");
 	}
 
+#if PG_VERSION_NUM >= 120000
+	if (vacuumFlags & VACOPT_SKIP_LOCKED)
+	{
+		appendStringInfoString(vacuumPrefix, "SKIP_LOCKED,");
+	}
+
+	if (vacuumParams.truncate != VACOPT_TERNARY_DEFAULT)
+	{
+		appendStringInfoString(vacuumPrefix,
+							   vacuumParams.truncate == VACOPT_TERNARY_ENABLED ?
```

Comment:
Minor: can't we make `0` -> `false`? Because these might show up in the logs, and `false` looks better.